src/components/ValidatedInput/ValidSelectAsync.scss

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2021 AccelByte Inc. All Rights Reserved.
+ * This is licensed software from AccelByte Inc, for limitations
+ * and restrictions contact your company contract manager.
+ */
+
+@import "src/styles/variables";
+
+.valid-select-async-input {
```

Comment:
i suggest to add `font-family: $font-family-primary` so that the font looks better in storybook

Reply:
updated, thanks